recipes/dask-yarn/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dask-yarn" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 043bb09dbc14d6aeb6460fc571c68ca623d2cedeaed26ba15bc8c6b396e39053
+
+build:
+  number: 0
+  entry_points:
+    - dask-yarn-worker=dask_yarn.cli.dask_yarn_worker:main
+    - dask-yarn-scheduler=dask_yarn.cli.dask_yarn_scheduler:main
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - dask >=0.18.0
+    - distributed >=1.22.0
+    - skein >=0.0.3
+    - backports.weakref  # [py2k]
+
+test:
+  imports:
+    - dask_yarn
+    - dask_yarn.cli
+
+about:
+  home: https://github.com/dask/dask-yarn
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: Deploy dask clusters on Apache YARN
+  description: |
+    Dask-Yarn deploys Dask on YARN clusters, as are found in traditional Hadoop
```

Comment:
Should perhaps be "*such* as are found"  I think I made this mistake a while ago and it has unfortunately propagated.